sbom.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
What's the meaning of this file? How was it generated, is it used?

Reply:
It's a software bill of materials (SBOM) generated with [CycloneDX](https://cyclonedx.org/)'s [Python tool](https://github.com/CycloneDX/cyclonedx-python). It's not used in this repository explicitly, but can be used by other tools like Dependency Track to get an overview of dependencies, vulnerabilities etc. 

(At SUB we're aiming at creating SBOMs for most of our projects and monitor them with an own instance of Dependency Track. I have to look into the dependencies, though, as they aren't fetched properly at this point which makes this file a bit pointless I admit. Nevertheless I think it's good practice to make 'cascading' dependencies a bit more transparent. Or isn't this the case? Is it sufficient to have a tool like dependabot in your repo? :thinking: )

Reply:
That is a sensible approach and I think it's better if we build our own dependency tracking infrastructure with a cronjob rather than depending on an external service. I am unfamiliar with CycloneDX, so maybe you can give me a short intro on how you use it at SUB.

Reply:
Sure, we can make an appointment if you like to!